### DISCUSSION TITLE
Just started the first course

### DISCUSSION THREAD

**Discussion by eyp** [OpenMined/courses#173] [Introduce Yourself]
Hi people, 

I'm Eduardo Yáñez from Spain, but living in Belgium. I work as a software engineer since 1996, and got my bachelor degree in computer engineer in 2000, but I like to keep on learning so I started the master's degree 2 years ago and just finished this month. My master's project has been related to Federated Learning (https://github.com/eyp/federated-learning-network), so I had to read a lot about it and met Openmined during the way. You have provided lot of resources to the community about privacy and deep learning that helped me with my project, so thank you.

Yesterday I started your new course, I guess I'll enjoy it. Learning different concepts and ideas is a good way to start the year.

See you all.

**Comment by yemikifouly** [comment]
Hello Eduardo! 👋 Welcome to the discussion board! I am really glad that we have helped you so far and I hope you enjoy the courses of the series very much! 🙂 How do you like the first course so far?

**Reply by eyp** [reply]
Thank you Yemi, I've just started, now in the middle of lesson 2, but I like it. I lile the idea of not being a technical course, I think it's a good idea because the concepts and foundations are really important to be able to develop more complex ideas from them.
